webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -132,4 +134,24 @@ describe('Test utils', () => {
         });
     });
   });
+  
+  it('Check that all test specs belong to a test suites', () => {
+    const testFolders = [];
+    for (const [, value] of Object.entries(constants.SUITES)) {
+      value.forEach(path => testFolders.push(path.split('/')[1]));
+    }
+    const getDirectories =   (src, callback) => {
```

Comment:
We should promisify this: 
```suggestion
const getDirectories = (src) =>
      new Promise((resolve, reject) => glob(src, (err, res) => err ? reject(err) : resolve(res)));
```